app/src/main/java/fi/bitrite/android/ws/persistence/MessageDao.java

```diff
@@ -148,7 +148,7 @@ private void saveMessage(SQLiteDatabase db, Message message) {
         cv.put("author_id", message.authorId);
         cv.put("date", DateConverter.dateToLong(message.date));
         cv.put("body", message.rawBody);
-        int status = (message.isNew ? 1 : 0) << MessageTable.STATUS_BIT_IS_NEW |
+        int status = (message.isNew ? 1 : 0) |
```

Comment:
I think this change is error prone. If one changes `MessageTable.STATUS_BIT_IS_NEW` then we no longer take care of that change here.

Reply:
I’m torn here. On the one hand, “YAGNI”. The status is as it is at the moment and we shouldn’t fix something prematurely that isn’t in use *yet*. On the other hand, I can see how that might slip through.

You decide. :)